components/Footer.tsx

```diff
@@ -0,0 +1,46 @@
+import Image from "next/image";
+
+export default function Footer(props) {
+    return (
+        <>
+        <footer>
+            <a
+            href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
+            target="_blank"
+            rel="noopener noreferrer"
+            >
+            Powered by{' '}
+            <Image src="/vercel.svg" alt="Picture of the product" width={200} height={200} className="logo"/>
+            </a>
+        </footer>
+
+        <style jsx>{`
```

Comment:
@duoctvd 
bỏ `<style jsx>` đi, dùng styled-component

Reply:
@bit-thuynt
Almost done
But this css (index.tsx) em vẫn đang tìm hiểu xem phải bỏ nó ở đâu
 https://gyazo.com/aa100d9c2bf3455b69540873798c3a0b

Reply:
@duoctvd Em sử dụng global style của styled-component https://styled-components.com/docs/api#createglobalstyle

Reply:
@duoctvd 
code tham khảo : https://github.com/vercel/next.js/blob/canary/examples/with-styled-components/pages/_app.js
Ở đây có nhiều ví dụ về styled-component, em có thể tham khảo
https://github.com/vercel/next.js/tree/canary/examples